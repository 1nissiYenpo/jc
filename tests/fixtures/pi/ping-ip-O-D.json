{"destination_ip": "192.168.120.164", "data_bytes": 56, "pattern": null, "destination": "192.168.120.164", "packets_transmitted": 5, "packets_received": 0, "packet_loss_percent": 100.0, "duplicates": 0, "time_ms": 4154.0, "responses": [{"type": "timeout", "timestamp": 1596585008.591169, "icmp_seq": 1}, {"type": "timeout", "timestamp": 1596585009.631169, "icmp_seq": 2}, {"type": "timeout", "timestamp": 1596585010.671173, "icmp_seq": 3}, {"type": "timeout", "timestamp": 1596585011.711167, "icmp_seq": 4}]}
