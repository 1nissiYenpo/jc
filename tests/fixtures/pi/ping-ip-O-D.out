PING 192.168.120.164 (192.168.120.164) 56(84) bytes of data.
[1596585008.591169] no answer yet for icmp_seq=1
[1596585009.631169] no answer yet for icmp_seq=2
[1596585010.671173] no answer yet for icmp_seq=3
[1596585011.711167] no answer yet for icmp_seq=4

--- 192.168.120.164 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 4154ms


